templates/apiary-hms-readwrite.json

```diff
@@ -62,6 +62,10 @@
         "name": "SNS_ARN",
         "value": "${sns_arn}"
      },
+     {
+      "name": "ATLAS_HIVE_SYNC",
+      "value": "${atlas_hive_sync}"
+     },
```

Comment:
wouldn't this also be needed in `apiary-hms-readonly.json`?

Reply:
I thought to send events to atlas, we ll only use readwrite endpoint. Correct me if I am wrong

Reply:
No, you are correct - sorry.